projects/plugins/crm/modules/portal/templates/single-invoice.php

```diff
@@ -36,18 +35,22 @@
 	box-shadow: none !important;
 }
 .zbs-back-to-invoices a:hover{
-  text-decoration:none;
+	text-decoration:none;
 }
 </style>
 
-<div class="alignwide zbs-site-main zbs-portal-grid<?php echo $show_nav?'':' no-nav' ?>">
-  <?php if ( $show_nav ) { ?>
-    <nav class="zbs-portal-nav"><?php echo $portal->render->portal_nav( $portal->get_endpoint( ZBS_TYPE_INVOICE ), false ); ?></nav>
-  <?php } ?>
-  <div class='zbs-portal-content zbs-portal-invoices-list'>
-    <div class='zbs-entry-content zbs-single-invoice-portal' style="position:relative;">
-      <?php $single_invoice_endpoint->single_invoice_html_output( $obj_id, true ); ?>
-    </div>
-  </div>
-  <div class="zbs-portal-grid-footer"><?php $portal->render->portal_footer(); ?></div>
+<div class="alignwide zbs-site-main zbs-portal-grid<?php echo $show_nav ? '' : ' no-nav'; ?>">
+	<?php if ( $show_nav ) { ?>
+		<nav class="zbs-portal-nav">
+			<?php
+			echo $portal->render->portal_nav( $portal->get_endpoint( ZBS_TYPE_INVOICE ), false ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			?>
+		</nav>
+	<?php } ?>
+	<div class='zbs-portal-content zbs-portal-invoices-list'>
+		<div class='zbs-entry-content zbs-single-invoice-portal' style="position:relative;">
```

Comment:
These two also use single quotes.

```suggestion
	<div class="zbs-portal-content zbs-portal-invoices-list">
		<div class="zbs-entry-content zbs-single-invoice-portal" style="position:relative;">
```